sql.txt

```diff
@@ -1,14 +1,30 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+-- SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country ASC Limit 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+-- SELECT * FROM Customers WHERE ContactName LIKE "A%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+-- SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+-- INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price)
+			VALUES 
+            	("Jade Leaf Matcha", 30, 2, "48 - 0.7 oz", 9),
+            	("Ummon-no-mukashi", 30, 2, "18 - 0.71 oz", 30),
+            	("Guayaki Yerba Mate", 30, 2, "24 - 16 oz", 16);
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+-- SELECT Price + 1.50 FROM Products WHERE ProductID = 1 OR ProductID = 2;
```

Comment:
in order to update a row in the database you need to use the `UPDATE` command.  The select command just return data from the database, but does not change it.
something along the lines of:
`UPDATE Products SET Price = Price + 1.50 WHERE ProductID IN (1,2);`